src/main/java/ch/njol/skript/conditions/CondMethodExists.java

```diff
@@ -16,7 +16,7 @@
  *
  * Copyright Peter Güttinger, SkriptLang team and contributors
  */
-package ch.njol.skript.test.runner;
```

Comment:
Shouldn't this stay as is?

Reply:
I was originally having moved these classes outside of the test runner, but only JUnit syntaxes should be in the test packages src/test so this is to be reverted. But CondMinecraftVersion in my opinion should be general access and not just the test runner.